runexporterapp.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export IRONIC_CONFIG=/etc/ironic/ironic.conf
+export FLASK_RUN_HOST=0.0.0.0
+export FLASK_RUN_PORT=9608
```

Comment:
I expect we'll want to be able to control this with a variable, too.

Reply:
can this be done in a follow-up?

Reply:
Why? Isn’t it a trivial one line change to accept an env var?